src/test/integration/tests/Deposit_Delegate_Redelegate_Complete.t.sol

```diff
@@ -56,6 +123,77 @@ contract Integration_Deposit_Delegate_Redelegate_Complete is IntegrationCheckUti
         // Fast forward to when we can complete the withdrawal
         cheats.roll(block.number + delegationManager.withdrawalDelayBlocks());
         staker.completeWithdrawalAsShares(withdrawals[0]);
+
+        check_Withdrawal_AsShares_Undelegated_State(staker, operator1, withdrawals[0], strategies, shares);
+
+        // 5. Delegate to a new operator
+        staker.delegateTo(operator2);
+        check_Delegation_State(staker, operator2, strategies, shares);
+        assertNotEq(address(operator1), delegationManager.delegatedTo(address(staker)), "staker should not be delegated to operator1");
+
+        // 6. Queue Withdrawal
+        withdrawals = staker.queueWithdrawals(strategies, shares);
+        withdrawalRoots = _getWithdrawalHashes(withdrawals);
+        check_QueuedWithdrawal_State(staker, operator2, strategies, shares, withdrawals, withdrawalRoots);
+
+        // 7. Complete withdrawal
+        // Fast forward to when we can complete the withdrawal
+        cheats.roll(block.number + delegationManager.withdrawalDelayBlocks());
+
+        // Complete all but last withdrawal as tokens
+        for (uint i = 0; i < withdrawals.length - 1; i++) {
+            staker.completeWithdrawalAsTokens(withdrawals[i]);
+        }
+
+        // Complete last withdrawal as shares
+        staker.completeWithdrawalAsTokens(withdrawals[withdrawals.length - 1]);
+        check_Withdrawal_AsTokens_State(staker, operator2, withdrawals[withdrawals.length - 1], strategies, shares);
+    }
+
+    function testFuzz_deposit_delegate_reDelegate_withAdditionalDepositBefore(uint24 _random) public {   
```

Comment:
I think the "additional deposit before" part of this test may have been removed from the rebases... can you add back?

Reply:
Still think it hasn't been added